_includes/head.html

```diff
@@ -7,5 +7,5 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
     <link type="text/css" rel="stylesheet" href="/application.css"/>
-    <script async src="https://analytics.umami.is/script.js" data-website-id="209b35e2-b972-4bf5-8a88-09cedbc740f8"></script>
+    <script async src="https://analytics.umami.is/script.js" data-website-id="209b35e2-b972-4bf5-8a88-09cedbc740f8" data-domains="ruby.uy"></script>
```

Comment:
right but this doesn't prevent that the Argentinian people that already got our script without this data attribute to push more events 🤪

Reply:
That it doesn't. I don't think there's a way to limit that with the current analytics config we have (free).
But at the very least they can pull that commit and stop sending analytics 😵‍💫